### DISCUSSION TITLE
Is ExternalName only available on nginx-plus ?

### DISCUSSION THREAD

**Discussion by biswanaths** [nginx/kubernetes-ingress#3772] [Q&A] [answered]
Hi, 

I was looking around the ExternalName feature for one of my use case. I just wanted to clarify whether if this feature is only available only for nginx-plus users ? 

The release note mentions this feature is only available for plus users https://www.nginx.com/blog/announcing-nginx-ingress-controller-for-kubernetes-release-1-5-0/#ExternalName 

I am not able to pull out any feature matrix comparing the features of nginx and nginx-plus wrt to controller. 

Thank you.

**Comment by vepatel** [comment, marked as answer]
Hi @biswanaths yes, `ExternalName` is only available to plus users:

- https://github.com/nginxinc/kubernetes-ingress/tree/v3.1.0/examples/custom-resources/externalname-services
- https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#upstream (See: `service`)

**Comment by ZeChArtiahSaher** [comment]
dumb
